tests/test_moisture_sensor.py

```diff
@@ -7,14 +7,21 @@
 
 class MoistureSensorTest(unittest.TestCase):
 
-    def setUp(self):
-        self.mock_adc = mock.Mock()
+    def test_moisture_50(self):
```

Comment:
The `50`'s out of date, but I think we can do something more descriptive, like `test_moisture_read_succeeds_when_adc_succeeds`.